roles/openshift_node_upgrade/tasks/rpm_upgrade.yml

```diff
@@ -12,3 +12,8 @@
 - name: Ensure python-yaml present for config upgrade
   package: name=PyYAML state=present
   when: not openshift.common.is_atomic | bool
+
+- name: Install Node service file
+  template:
+    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
+    src: "{{ openshift.common.service_type }}-node.service.j2"
```

Comment:
Ok, lets leave it where you have it but we need to reload systemd do we not? If so something like
```
- name: Install Node service file
  template:
    dest: "/etc/systemd/system/{{ openshift.common.service_type }}-node.service"
    src: "{{ openshift.common.service_type }}-node.service.j2"
  register: node_unit

- name: Reload systemd units
  command: systemctl daemon-reload
  when: node_unit | changed
```